src/js/state/Layout/reducer.js

```diff
@@ -4,7 +4,10 @@ import type {LayoutState, LayoutAction} from "./types"
 
 const init: LayoutState = {
   rightSidebarIsOpen: false,
-  rightSidebarWidth: 450
+  rightSidebarWidth: 450,
+  leftSidebarIsOpen: false,
+  leftSidebarWidth: 350,
+  investigationView: "linear"
```

Comment:
We'll need to add a migration for this state change. Maybe all the layouts will be initialized with the old "view" global setting.